js/libs/form.js

```diff
@@ -655,6 +655,34 @@ Fliplet.Widget.instance('form-builder', function(data) {
           $vm.saveProgress();
         }, 10);
 
+        function validateCheckboxValue(value, options) {
+          value = _.isArray(value) ? value : [value];
+
+          value = value.map(function(elem) {
+            return elem.trim();
```

Comment:
I don't think this part is necessary. I vaguely remembered @squallstar suggested using `.trim()` because there was a `.split(",")` operation happening and we want to make sure the values are trimmed. We're not doing that any more so this part shouldn't be necessary.

@squallstar Thoughts?

Reply:
@tonytlwu if we're not using `split` then we shouldn't `trim`, that's correct.

Reply:
fixed